src/main/java/com/commercetools/project/sync/CliRunner.java

```diff
@@ -40,7 +40,8 @@
   static final String SYNC_MODULE_OPTION_DESCRIPTION =
       format(
           "Choose one of the following modules to run: \"%s\", \"%s\", \"%s\", \"%s\", \"%s\", \"%s\", \"%s\", \"%s\", \"%s\" or \"%s\".",
-          ArrayUtils.add(SyncModuleOption.getSyncOptionValues(), SYNC_MODULE_OPTION_ALL));
+          (Object[])
```

Comment:
why are you casting here?

Reply:
Good question. The reason is this: method `format` takes varargs as a param. If I don't specify, the method is confused if what I passed should be 1 argument (`ArrayUtils.add(SyncModuleOption.getSyncOptionValues(), SYNC_MODULE_OPTION_ALL)`) or should it be varargs. Thus the casting so there's no doubt that I want to pass it as varargs.

Reply:
thanks for the explanation